### PR TITLE
add permissions & viewMode to loadConfig

### PR DESCRIPTION
**Pull request by ranbreuer**

### CODE REVIEW

File: test/models.spec.ts [hunk 1]
```diff
@@ -187,6 +189,74 @@ describe('Unit | Models', function () {
...
+        }
+      };
+
+      // Act
+      const errors = models.validateReportLoad(testData.load);
+
+      // Assert
+      testForExpectedMessage(errors, viewModeInvalidMessage);
+    });
   });
```
Comment (ali-hamud) [inline test/models.spec.ts]: you may want to add two additional tests for success cases (to see that they don't return invalid message)

File: typings.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {},
   "globalDependencies": {
-    "jasmine": "registry:dt/jasmine#2.2.0+20160621224255",
-    "jsen": "registry:dt/jsen#0.0.0+20160328204314",
-    "karma-jasmine": "registry:dt/karma-jasmine#0.0.0+20160316155526"
+    "jasmine": "registry:dt/jasmine",
+    "jsen": "registry:dt/jsen",
+    "karma-jasmine": "registry:dt/karma-jasmine"
   }
```
Comment (ali-hamud) [inline typings.json]: don't we need to add minimum version?

File: src/models.ts [hunk 3]
```diff
@@ -346,3 +346,15 @@ export interface ISelection {
   regions: IIdentityValue<IEqualsDataReference | IBetweenDataReference>[];
   filters: (IBasicFilter | IAdvancedFilter)[];
 }
+
```
Comment (ali-hamud) [inline src/models.ts]: you are not using these exports.
do you want to add them to load config?

File: src/models.ts [hunk 4]
```diff
@@ -346,3 +346,15 @@ export interface ISelection {
...
+export enum Permissions {
```
Comment (ArinaHantsis) [inline src/models.ts]: We might incorporate permissions into ViewMode setting as it basically controls the view mode capabilities the report loads with.

File: test/models.spec.ts [hunk 5]
```diff
@@ -187,6 +189,74 @@ describe('Unit | Models', function () {
...
+      };
+
+      // Act
+      const errors = models.validateReportLoad(testData.load);
+
+      // Assert
+      testForExpectedMessage(errors, permissionsInvalidMessage);
+    });
+
+    it(`should return errors with one containing message '${viewModeInvalidMessage}' if permissions is not a number`, function () {
```
Comment (ArinaHantsis) [inline test/models.spec.ts]: viewMode

File: src/models.ts (hunk 3 above)
Comment (ali-hamud) [inline src/models.ts]: I think that Permissions should be passed in jwt token not here.
please re-think it.

File: package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-models",
-  "version": "0.10.1",
+  "version": "0.10.2",
```
Comment (noanu) [inline package.json]: The version should be 0.10.3 . The current version in DEV branch is 0.10.2 (not sure how this diff was created , but something looks wrong)

File: typings.json (hunk 2 above)
Comment (ranbreuer) [inline typings.json]: this is not minimum version, but specific version

**Pull request merged**